x/inflation/keeper/hooks_test.go

```diff
@@ -66,6 +66,7 @@ func (suite *KeeperTestSuite) TestPeriodChangesAfterEpochEnd() {
 		name            string
 		currentPeriod   int64
 		height          int64
+		epochIdentifier string
 		skippedEpochs   uint64
 		enableInflation bool
 		changes         bool
```

Comment:
Let's rename the test to reflect that it's testing periods and skipped epochs. Would also adjust the name of the field `changes`, e.g. `periodChanges`